packages/riipen-ui/src/components/Button.jsx

```diff
@@ -65,13 +65,13 @@ const Button = props => {
         <span className={clsx("label")}>
           {IconStart && (
             <span className={clsx("icon", "icon-start")}>
-              <IconStart />
+              {typeof IconStart === "function" ? <IconStart /> : IconStart}
```

Comment:
Yeah definitely add tests for the cases when `IconStart` and End are functions and when they're not.